rbac/server/api/main.py

```diff
@@ -67,6 +67,15 @@ def load_config(app):
     app.config.AES_KEY = get_config("AES_KEY")
     app.config.AIOHTTP_CONN_LIMIT = int(get_config("AIOHTTP_CONN_LIMIT"))
     app.config.AIOHTTP_DNS_TTL = int(get_config("AIOHTTP_DNS_TTL"))
+    app.config.API_SECURITY = [{"authToken": []}]
+    app.config.API_SECURITY_DEFINITIONS = {
+        "authToken": {
+            "type": "apiKey",
+            "in": "header",
+            "name": "Authorization",
+            "description": 'Paste your auth token and do not forget to add "Bearer " in front of it',
```

Comment:
Need to update this description, we don't need to put Bearer in front. Just mention that the user needs to just paste their auth token in only.